pythonosc/osc_message_builder.py

```diff
@@ -78,6 +78,8 @@ def add_arg(self, arg_value, arg_type=None):
         arg_type = self.ARG_TYPE_TRUE
       elif builtin_type == builtins.bool and not arg_value:
         arg_type = self.ARG_TYPE_FALSE
+      else:
+        raise ValueError('Infered arg_value type is not supported')
```

Comment:
The add_arg method can also now raise a TypeError if you use isinstance, can you add this to the documentation of this method please?

https://docs.python.org/3/library/functions.html#isinstance